### DISCUSSION TITLE
dumi执行npm run build,忽略校验ts，加什么参数

### DISCUSSION THREAD

**Discussion by aifetch** [umijs/dumi#2128] [Q&A]
dumi执行npm run build,忽略校验ts，加什么参数

**Comment by PeachScript** [comment]
框架不提供忽略类型错误的配置，既然用了 TypeScript 保证类型正确应当是首选，如果不想要类型检查可以改写 `.js` 文件，或者使用 `ts-nocheck` 做指定文件的忽略

dumi 脚手架中源码构建脚本对应的工具是 father，后续源码构建相关问题请到 father 仓库中提交 Discussion：https://github.com/umijs/father
